src/main.cpp

```diff
@@ -1,227 +1,49 @@
-//by Alexander Prosser
-
+// snake
```

Comment:
**_IT'S SNEK!!_**

Reply:
xD, but we need to add a propper header anyway